tests/helpers/skipif.py

```diff
@@ -59,14 +61,49 @@ def test_any_func(...):
         return dict(condition=False, reason="Conditions satisfied, going ahead with the test.")
 
     reasons = [rs for cond, rs in zip(conditions, reasons) if cond]
-    return dict(condition=any(conditions), reason=f"Required: [{' + '.join(reasons)}]",)
+    return dict(
+        condition=any(conditions),
+        reason=f"Required: [{' + '.join(reasons)}]",
+    )
+
+
+class LightSkipif(MarkDecorator):
```

Comment:
Rename to `LitSkipIf` or `LitSkipif`?

Also, no need to subclass `MarkDecorator`

```suggestion
class LightSkipif:
```